src/components/LandingSection/LandingSection.js

```diff
@@ -15,17 +17,48 @@ const LandingHeader = styled.h1`
   font-family: ${fonts.families.headerFont};
   font-weight: ${fonts.weights.regular};
   width: 80%;
-  word-break: break-all;
+`;
+
+const ChangeLocaleWrapper = styled.div`
+  position: fixed;
+  top: ${spacing.xlSize};
+  left: -${spacing.xxxxlSize};
+
+  transform: ${({ isNavActive }) =>
+    isNavActive ? 'none' : 'translateX(120px)'};
```

Comment:
`translateX(0)` better would be `none` in terms of performance and rendering part of it. We only change the tranfsorm instead of "recreate" it